src/k2/assignmentManager/AssignmentManager.cpp

```diff
@@ -98,8 +96,19 @@ AssignmentManager::handleAssign(dto::AssignmentCreateRequest&& request) {
         partition.endpoints.insert(rdma_ep->url);
     }
 
+    String cpoEP = "";
```

Comment:
The CPO endpoint should come as an auto-rrdma, not tcp/rdma directly. Otherwise, we won't be able to restart the CPO.

Reply:
Add transport helper for ep string selection